ListItem/Alphi Added font to assets; Edited Shadow; Added new Color; Combined with new code.

aahenry51 opened the pull request.

aahenry51 commented:
This is the build-in font.

![oldfont](https://user-images.githubusercontent.com/79045258/159771063-71e65c31-b956-457a-b56e-3de1b2ee1125.png)

This is the Poppins font.

![Newfont](https://user-images.githubusercontent.com/79045258/159771275-186928db-c7eb-4768-8004-e12079e68f50.png)

Nap1400 reviewed (approved):
I need this font for the StepsProgress component.

aahenry51 commented:
![Shadow](https://user-images.githubusercontent.com/79045258/159805727-1d7b6d8b-26be-4c67-bf71-e96a466339c9.png)

Integrated my code, Changed shadow properties for list item, Changed Border radius, Added a new color.

carizzuti reviewed (approved):
Color looks good and UI is clean

raynesmike commented on src/components/listItem/ListItem.js:
```diff
@@ -1,48 +1,45 @@
-  subTitleStyle,
-  iconContainerStyle,
-  ...rest
+  MaxTitleLength,
```
camelCase this one

aahenry51 replied:
Fixed

lasharela commented:
<img width="479" alt="image" src="https://user-images.githubusercontent.com/2189316/159812486-b8021cbe-5ad9-4a79-83da-1599537cfc3c.png">
Subtitle text doesn't match . It's regular in figma

aahenry51 commented:
> <img alt="image" width="479" src="https://user-images.githubusercontent.com/2189316/159812486-b8021cbe-5ad9-4a79-83da-1599537cfc3c.png">
> 
> Subtitle text doesn't match . It's regular in figm
Both the title and subtitle are regular in figma.

leo-he-1 reviewed (approved):
I get errors when I run this branch. Not sure if it's a problem on my end.
![image](https://user-images.githubusercontent.com/70768391/160207135-3d56f4a0-ea38-4867-a1a7-97e65089aeda.png)

raynesmike reviewed (approved):
I approve with all these changes

aahenry51 commented:
![image](https://user-images.githubusercontent.com/79045258/160246942-ddc49e6b-41ed-4b3a-a596-7c1b8b1bd680.png)

aahenry51 commented:
![image](https://user-images.githubusercontent.com/79045258/160247001-134db080-6df6-44b3-b814-4a2a86dd1299.png)
